### ISSUE TITLE
 Async APIs fails with job query timed out error, after setting empty/invalid timeout config

### ISSUE THREAD

**Issue by sureshanaparti** [apache/cloudstack-cloudmonkey#102]
Async APIs fails with job query timed out error, after after setting empty timeout (Ex: '_set timeout_ ' or '_set timeout test_ 'cmd), which is already prompted as invalid. After setting to valid value,  Async APIs starts working.

```
(cloudcmk) 🐱 > set timeout 
Error caught while setting timeout: strconv.Atoi: parsing "": invalid syntax
(cloudcmk) 🐱 > 
(cloudcmk) 🐱 > migrate virtualmachinewithvolume virtualmachineid=3a879829-466a-451a-bb2d-a2897349a9d9 hostid=d19ac1e9-990d-404d-b60c-a7b75cf7836c  migrateto[0].volume=c01302c0-41e4-4fbe-876e-13c3493ad8f6 migrateto[0].pool=cae2c100-cfd8-47f1-aed5-3442b289ec85 migrateto[1].volume=9f2f170a-6650-4750-bd9c-76fca2ae70cb migrateto[1].pool=df3aba77-9ff6-3c91-9062-f2d79686672e
🙈 Error: async API job query timed out
(cloudcmk) 🐱 >
(cloudcmk) 🐱 > sync storagepool id=10797937-8916-35b7-8ceb-cc1070b1e42f 
🙈 Error: async API job query timed out
(cloudcmk) 🐱 > 
(cloudcmk) 🐱 > set timeout 3600
(cloudcmk) 🐱 > 
(cloudcmk) 🐱 > sync storagepool id=10797937-8916-35b7-8ceb-cc1070b1e42f 
{
  "accountid": "0cdf9483-2123-11ec-b664-1e000d000227",
  "cmd": "org.apache.cloudstack.api.command.admin.storage.SyncStoragePoolCmd",
  "completed": "2021-09-30T12:17:54+0000",
  "created": "2021-09-30T12:17:54+0000",
  "jobid": "8c749b80-1e53-4f76-860a-8b82bd7fcfc0",
  "jobprocstatus": 0,
  "jobresult": {
    "errorcode": 431,
    "errortext": "SyncStoragePool API is currently supported only for storage type of datastore cluster"
  },
  "jobresultcode": 530,
  "jobresulttype": "object",
  "jobstatus": 2,
  "userid": "0ce1659d-2123-11ec-b664-1e000d000227"
}
🙈 Error: async API failed for job 8c749b80-1e53-4f76-860a-8b82bd7fcfc0
(cloudcmk) 🐱 >
(cloudcmk) 🐱 > set timeout test
Error caught while setting timeout: strconv.Atoi: parsing "test": invalid syntax
(cloudcmk) 🐱 > 
(cloudcmk) 🐱 > sync storagepool id=10797937-8916-35b7-8ceb-cc1070b1e42f 
🙈 Error: async API job query timed out
(cloudcmk) 🐱 >
```

**Comment by rhtyd** [comment]
Can you check if these are regressions/blockers against previous releases warranting a new RC or just bugs?

**Comment by Pearl1594** [comment]
Not a regression - has been the behvior from before:
```
[root@ref-trl-1795-k-M7-pearl-dsilva-mgmt1 ~]# cmk
Apache CloudStack 🐵 CloudMonkey 6.0.0
Report issues: https://github.com/apache/cloudstack-cloudmonkey/issues

(dom) SBCM5> > 
(dom) SBCM5> > set timeout test
Error caught while setting timeout: strconv.Atoi: parsing "test": invalid syntax
(dom) SBCM5> > 
(dom) SBCM5> > create project name=testProj displaytext=testProj
🙈 Error: async API job query timed out
(dom) SBCM5> >  


```

**Comment by sureshanaparti** [comment]
> Can you check if these are regressions/blockers against previous releases warranting a new RC or just bugs?

Not a regression @rhtyd

**Comment by rhtyd** [comment]
Fixed by #103
